src/CheckoutSdk/Disputes/GetDisputesRequest.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using System;
+
+namespace Checkout.Disputes
+{
+    /// <summary>
+    /// Defines a <see cref="GetDisputesRequest"/> for getting a list of all disputed payments.
+    /// </summary>
+    public class GetDisputesRequest
+    {
+        /// <summary>
+        /// Creates a new get disputes request.
+        /// </summary>
+        /// <param name="limit">The numbers of results to return. 1 .. 250, default is 50.</param>
+        /// <param name="skip">The number of results to skip, default is 0.</param>
+        /// <param name="from">The ISO-8601 date and time from which to filter disputes, based on the dispute's last_update field.</param>
+        /// <param name="to">The ISO-8601 date and time until which to filter disputes, based on the dispute's last_update field.</param>
+        /// <param name="id">The unique identifier of the dispute.</param>
+        /// <param name="statuses">One or more comma-separated statuses. This works like a logical OR operator.</param>
+        /// <param name="paymentId">The unique identifier of the payment.</param>
+        /// <param name="paymentReference">An optional reference (such as an order ID) that you can use later to identify the payment.</param>
+        /// <param name="paymentArn">The acquirer reference number (ARN) that you can use to query the issuing bank.</param>
```

Comment:
```suggestion
        /// <param name="paymentArn">The acquirer reference number (ARN) that you can use to query the issuing bank.</param>
        /// <param name="thisChannelOnly">If true, only returns disputes of the specific channel that the secret key is associated with. Otherwise, returns all disputes for that business</param>
```
Add xml-doc comment for `thisChannelOnly` parameter

Reply:
added